### ISSUE TITLE
Couldn't apply any add-on config

### ISSUE THREAD

**Issue by Mfungorn** [mozilla-iot/gateway#2354]
When I add any add-on and try to configure it I see an Error message below
![image](https://user-images.githubusercontent.com/41123969/70371978-2d633600-190c-11ea-85f2-cee0af7237b2.png)
![image](https://user-images.githubusercontent.com/41123969/70372014-7a470c80-190c-11ea-8f60-129a2d4e7888.png)

Then I have several WebSocket Errors in Console:
![image](https://user-images.githubusercontent.com/41123969/70372059-06593400-190d-11ea-9085-b02ec1d27559.png)

After that gateway tries to ping and gets "Gateway unreachable", but gateway is active and work properly
![image](https://user-images.githubusercontent.com/41123969/70372032-c3975c00-190c-11ea-9d3b-46f2b6a4aa3f.png)

**Comment by Mfungorn** [comment]
My Current version of Gateway is 0.10.0
Last update: Thu Nov 14 2019 09:31:26 GMT+0700

**Comment by Mfungorn** [comment]
Recent internal logs:
```
2019-12-07 16:12:42.254 INFO   : Loading add-on: telegram-sender-adapter
2019-12-07 16:12:42.580 INFO   : Loading add-on: thing-url-adapter
2019-12-07 16:12:42.939 INFO   : Loading add-on: virtual-things-adapter
2019-12-07 16:12:43.905 INFO   : PageKite connected!
2019-12-07 16:12:43.913 INFO   : Starting certificate renewal.
2019-12-07 16:12:44.027 INFO   : Certificate not yet due for renewal.
2019-12-07 16:12:44.914 INFO   : email-sender: Opening database: /home/pi/.mozilla-iot/config/db.sqlite3
2019-12-07 16:12:45.059 INFO   : telegram-sender: Opening database: /home/pi/.mozilla-iot/config/db.sqlite3
2019-12-07 16:12:45.243 INFO   : thing-url: Opening database: /home/pi/.mozilla-iot/config/db.sqlite3
2019-12-07 16:12:45.275 INFO   : virtual-things: Opening database: /home/pi/.mozilla-iot/config/db.sqlite3
2019-12-07 16:12:47.104 INFO   : email-sender: Loading add-on email-sender-adapter from /home/pi/.mozilla-iot/addons/email-sender-adapter
2019-12-07 16:12:47.160 INFO   : telegram-sender: Loading add-on telegram-sender-adapter from /home/pi/.mozilla-iot/addons/telegram-sender-adapter
2019-12-07 16:12:47.261 INFO   : virtual-things: Loading add-on virtual-things-adapter from /home/pi/.mozilla-iot/addons/virtual-things-adapter
2019-12-07 16:12:47.353 INFO   : thing-url: Loading add-on thing-url-adapter from /home/pi/.mozilla-iot/addons/thing-url-adapter
2019-12-07 16:12:48.085 INFO   : thing-url: Starting mDNS discovery
2019-12-07 16:12:48.508 INFO   : Checking for add-on updates...
2019-12-07 16:12:50.328 INFO   : Finished updating add-ons
```

**Comment by mrstegeman** [comment]
The gateway crashed while restarting the add-on. This was fixed by mozilla-iot/gateway-addon-ipc-schema#4. It doesn’t always happen, so you can just keep trying until it works.
